templates/dm_vm/dm_vm.py

```diff
@@ -125,9 +124,23 @@ def install(self):
         if image == 'zero-os':
             if not self.data['ztIdentity']:
                 self.data['ztIdentity'] = vm.schedule_action('generate_identity').wait(die=True).result
-            url = IPXEURL.format(version, self.data['mgmtNic']['id'], self.data['ztIdentity'])
-            vm.schedule_action('update_ipxeurl', args={'url': url}).wait(die=True)
-
+            kernel_args = [
+                {
+                    'name': 'development',
```

Comment:
why is development hard coded ?

Reply:
Because it was already hard-coded before so I kept it the same. I think we kept development in the there by default to make it easy to debug if things go wrong, until everything is stable.

Reply:
hum since now we can configure the args from the template itself, I think we should not hard code it at this level. If any service using a VM needs to boot with development still it's stable enough, he can just put the development flag there.